src/librustdoc/clean/mod.rs

```diff
@@ -1694,22 +1737,135 @@ impl<'a, 'tcx> Clean<Generics> for (&'a ty::Generics,
     }
 }
 
+// The point is to replace bounds with types.
```

Comment:
Can you write doc comments for the functions you added? For example, this could be something like "Extract the types from a function's parameters and generics" with maybe a note about trait bounds since you've already added the comment for it.

Reply:
Ah sure.